src/translations/en.json

```diff
@@ -104,8 +120,8 @@
         "on": "[%key:state::binary_sensor::gas::on%]"
       },
       "opening": {
-        "off": "Closed",
```

Comment:
Leave these as they were and have the other types reference `"opening"`. Just a nitpick, but I'd rather have the specific types reference the more generic type.

Reply:
Agreed!